tutorials/poc-spread-onto-multiple-clusters.md

```diff
@@ -15,6 +15,61 @@ You can check connected clusters using kubectl:
     kind-workload-1   1.22          workload-1                    True    True   False     kubernetes.io/metadata.name=kind-workload-1,nova.elotl.co/cluster.novacreated=false,nova.elotl.co/cluster.provider=kind,nova.elotl.co/cluster.version=1.22
     kind-workload-2   1.22          workload-2                    True    True   False     kubernetes.io/metadata.name=kind-workload-2,nova.elotl.co/cluster.novacreated=false,nova.elotl.co/cluster.provider=kind,nova.elotl.co/cluster.version=1.22
 
+Before we start, let's talk about how Nova schedules Kubernetes workloads. On the lower level, Nova looks for a particular annotation on the k8s resource. The annotation contains a target cluster name, or, in the spread scheduling scenario, a list of target cluster names.
+Let's see it with a basic example:
+
+```shell
+    $ cat <<EOF > sample-spread-scheduling/busybox.yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: busybox
+  namespace: default
+  annotations:
+    nova.elotl.co/spread-onto: "kind-workload-1,kind-workload-2"
+  labels:
+    app: busybox
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: busybox
+  template:
+    metadata:
+      labels:
+        app: busybox
+    spec:
+      containers:
+      - name: busybox
+        image: busybox:stable
+        command:
+        - sleep
+        - inf
+EOF
+```
+Now, we can watch busybox deployment reaching 2 available replicas in the Nova Control Plane:
```

Comment:
Shouldn't we have a step here to apply the above yaml?

Reply:
good point!